boards/shared/Inc/Io/Io_SharedFreqOnlyPwmInput.h

```diff
@@ -34,15 +34,15 @@ struct FreqOnlyPwmInput *Io_SharedFreqOnlyPwmInput_Create(
  * @return The frequency for the given PWM input
  */
 float Io_SharedFreqOnlyPwmInput_GetFrequency(
-    struct FreqOnlyPwmInput *pwm_input);
+    const struct FreqOnlyPwmInput *pwm_input);
```

Comment:
I understand that I'm not supposed to use `const` in function prototypes. If I don't add a `const` in front of  `struct FreqOnlyPwmInput *pwm_input);`  I get a conflicting types error. I think in this case we are passing the value by reference so we should be ok. @thekenu

Reply:
You're getting it confused. There are two const you can add

```
const struct FreqOnlyPwmInput* const pwm_input
```
The second one is the redundant one. The one you added is fine :v: